requirements.txt

```diff
@@ -1,3 +1,3 @@
 numpy
 onnx
-onnxruntime
\ No newline at end of file
+# onnxruntime
```

Comment:
I'm commenting this out since there maybe use of onnxruntime-gpu instead some times